oak_remote_attestation_verification/tests/verifier_tests.rs

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use oak_remote_attestation_verification::{
+    proto::oak::attestation::v1::{
+        attestation_results::Status, Endorsements, Evidence, ReferenceValues,
+    },
+    verifier::verify,
+};
+
+#[test]
+fn verify_fails_with_empty_args() {
+    let evidence = Evidence::default();
+    let endorsements = Endorsements::default();
+    let reference_values = ReferenceValues::default();
+
+    let r = verify(&evidence, &endorsements, &reference_values);
+
+    assert!(Status::from_i32(r.status) == Some(Status::GenericFailure));
```

Comment:
This does not look particularly more type safe -- I was hoping that the status would be represented as enum?

Reply:
Reading https://github.com/tokio-rs/prost#enumerations it seems there should be an accessor method (but I haven't checked).

Reply:
Yes makes sense. Will address in next round.